app/src/renderer/routes.js

```diff
@@ -85,6 +85,11 @@ export default [
     name: `network`,
     component: require(`./components/network/PageNetwork`).default
   },
+  {
+    path: `/block/:height`,
```

Comment:
curious about why isn't this `/blocks/:height` ?

Reply:
mistake. good catch.